frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -163,10 +163,15 @@ export function FieldValuesWidgetInner({
   const [isExpanded, setIsExpanded] = useState(false);
   const dispatch = useDispatch();
 
+  const isSingleValueSearch = valuesMode === "search" && !multi;
```

Comment:
Should we take `disableSearch` into account here?

```suggestion
  const isSingleValueSearch = valuesMode === "search" && !multi && !disableSearch;
```